velox/functions/prestosql/tests/GreatestLeastTest.cpp

```diff
@@ -86,6 +127,78 @@ TEST_F(GreatestLeastTest, greatestDouble) {
       {100, 1.1, 1});
 }
 
+TEST_F(GreatestLeastTest, leastDecimal) {
```

Comment:
Are these 2 tests using any methods of GreatestLeastTest (without the new decimal additions) ? If no, then we could create a GreatestLeastDecimalTest class (in this same file) and derive the tests from it.

Reply:
Thanks for the suggestion, created another class for decimal tests.